### ISSUE TITLE
Eclipse Ant provision

### ISSUE THREAD

**Issue by jgsuess** [de-jcup/eclipse-asciidoctor-editor#374]
Could you expose some or all of your execution using ant tasks in eclipse-platform [Ant Extension Points](https://help.eclipse.org/latest/topic/org.eclipse.platform.doc.isv/reference/extension-points/org_eclipse_ant_core_antTasks.html)? This would allow running all the ASCIIDOCTOR beauty with none of the hassle in a CI workflow!

**Comment by de-jcup** [comment]
With #371 the build will be changed to maven tycho build - so CI workflow will become very easy and eclipse parts can be build completely. Or did I missunderstood your request/question ?

**Comment by jgsuess** [comment]
Yes, you have not understood this correctly. What I had hoped for was was that you could integratethe eclipse extension Point saw that the eclipse extension point for Ant tasks, sothat asciidoctor could be called from within eclipse's Ant facility directly.

**Comment by de-jcup** [comment]
Okay. I understand that you want to use the ANT eclipse plugin to trigger asciidoc actions.
So far so good.

But...
The eclipse asciidoctor editor is designed for developers to edit .adoc files, get an overview etc. For example
- Outline view
- Preview
- Call hierarchy
- Error markers
- Syntax highlighting
- .. more UI staff..

So what could you want to do with the editor plugin via ant?
- render an asciidoc file
- ... ?

The asciidoctor project does already provide multiple plugins/extensions. 
For ANT there is a specified plugin/extension available at https://github.com/asciidoctor/asciidoctor-ant 

This works in CLI and should also work in eclipse as well.

Is there a real benefit to use the editor plugin for ant to render?

**Comment by jgsuess** [comment]
That's right, but we are using ant + eclipse as a DevOps environment for model-driven operations. If we had your tooling available from ant, all the phantastic output could be rendered in our toolchains. High-quality, automatic documentation on every build, after every model-driven run. Given how many features the tool has, even a part would give us a dramatic boost!
